docs/user/adding-glean-to-your-project.md

```diff
@@ -45,8 +46,39 @@ For example, if version *0.34.2* is used, then the include directive becomes:
 apply from: 'https://github.com/mozilla/glean/raw/v0.34.2/glean-core/android/sdk_generator.gradle'
 ```
 
+### Adding new metrics
+
+All metrics that your project collects must be defined in a `metrics.yaml` file. 
+This file should be at the root of the module (the same directory as the `build.gradle` file you updated). 
+The format of that file is documented [here](https://mozilla.github.io/glean_parser/metrics-yaml.html).
```

Comment:
`here` is not a good click target. In addition it's used more often below, making it non-explanatory.

(I'm currently not coming up with a better wording though)